### ISSUE TITLE
 installation: Wheel not seen by tensorboard

### ISSUE THREAD

**Issue by jannisborn** [GT4SD/gt4sd-core#146]
**Describe the bug**
Wheel gets successfully installed via conda but is not seen by all pip packages

**To Reproduce**
Follow GPU or CPU installation

**Expected behavior**

**Screenshots**
```sh
  File "/u/jab/code/gt4sd-core/src/gt4sd/cli/inference.py", line 35, in <module>
    from ..algorithms.registry import ApplicationsRegistry
  File "/u/jab/code/gt4sd-core/src/gt4sd/algorithms/__init__.py", line 28, in <module>
    from .conditional_generation.guacamol.core import (  # noqa: F401
  File "/u/jab/code/gt4sd-core/src/gt4sd/algorithms/conditional_generation/guacamol/__init__.py", line 26, in <module>
    from .core import (
  File "/u/jab/code/gt4sd-core/src/gt4sd/algorithms/conditional_generation/guacamol/core.py", line 28, in <module>
    from ....training_pipelines.core import TrainingPipelineArguments
  File "/u/jab/code/gt4sd-core/src/gt4sd/training_pipelines/__init__.py", line 30, in <module>
    import pkg_resources
  File "/u/jab/.conda/envs/gt4sd/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3242, in <module>
    @_call_aside
  File "/u/jab/.conda/envs/gt4sd/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3226, in _call_aside
    f(*args, **kwargs)
  File "/u/jab/.conda/envs/gt4sd/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3255, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/u/jab/.conda/envs/gt4sd/lib/python3.7/site-packages/pkg_resources/__init__.py", line 570, in _build_master
    return cls._build_from_requirements(__requires__)
  File "/u/jab/.conda/envs/gt4sd/lib/python3.7/site-packages/pkg_resources/__init__.py", line 583, in _build_from_requirements
    dists = ws.resolve(reqs, Environment())
  File "/u/jab/.conda/envs/gt4sd/lib/python3.7/site-packages/pkg_resources/__init__.py", line 772, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'wheel>=0.26' distribution was not found and is required by tensorboard
```

**Comment by georgosgeorgos** [comment]
We do not have this issue installing in standard (not editable) mode.

**Comment by jannisborn** [comment]
I nailed this issue down, it's not related to GPU installation but it was essentially introduced in v0.51.0 when `tensorflow` was added to the `setup.cfg`. 
At the core of this is a version conflict between lightning (requiring a recent tensorboard) and tensorflow (requiring an old tensorboard). The easiest would be to relax the tensorflow dependency, one of the few packages that is pinned to exact version (2.1.0) due to MoLeR.

The workaround I found is to remove tensorflow from the `setup.cfg`, I will open a PR
